### PR TITLE
Addition of four more posters

### PR DESCRIPTION
**Pull request by Tofsel**
## About the PR
Changed the poster description from the one I made (HarMoney) just a little bit, and added four more to have more variety into the server. All of them were done by me (Denetth).

## Technical details
Added Missing Poster, UNMC Advert and a variation of it, Safety Lizard Poster inspired by some other Safety Moth posters. Also changed the description for HarMoney to be slightly more representative of Harmony Station.

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**

:cl:
- add: Added missing poster of a well known individual
- add: Added UNMC ad poster
- add: Added variation of the same UNMC poster
- add: Added Safety Lizz poster
- tweak: Changed HarMoney poster description

### CODE REVIEW

File: Resources/Textures/_Harmony/Structures/Wallmounts/posters.rsi/harmony_poster17_legit.png
Comment (FluffMe) [inline Resources/Textures/_Harmony/Structures/Wallmounts/posters.rsi/harmony_poster17_legit.png:1]: image

File: Resources/Textures/_Harmony/Structures/Wallmounts/posters.rsi/harmony_poster18_legit.png
Comment (FluffMe) [inline Resources/Textures/_Harmony/Structures/Wallmounts/posters.rsi/harmony_poster18_legit.png:1]: image

File: Resources/Prototypes/_Harmony/Entities/Structures/Wallmounts/Signs/posters.yml [hunk 1]
```diff
@@ -148,3 +148,44 @@
...
+- type: entity
+  parent: PosterBase
+  id: PosterHarmonyMuhrinePropaganda
+  name: "JOIN THE UNMC! OORAH!"
+  description: "An advert showcasing the multiple succesful operations the UNMC has been involved with, inviting to the viewer to enlist themselves to the Marines."
+  components:
+    - type: Sprite
+      sprite: _Harmony/Structures/Wallmounts/posters.rsi
+      state: harmony_poster19_legit
+
```
Comment (FluffMe) [inline Resources/Prototypes/_Harmony/Entities/Structures/Wallmounts/Signs/posters.yml:191]:
```suggestion
      
```

No EOF empty line

**Review by FluffMe — COMMENTED** [review]
* Added screenshots for @KeldWolf final review

**Comment by FluffMe** [comment]
If anyone merges this besides me, please merge with squash due to non-descriptive commit messages.

**Review by KeldWolf — APPROVED** [review]
Changes are made in the right places and with #Harmony comments where shared. Looks good. Approved.

**Pull request merged**